test/exmg-snackbars.spec.ts

```diff
@@ -0,0 +1,55 @@
+import {showSnackBar} from '../exmg-snackbars';
+
+const {assert} = chai;
+
+suite('<exmg-snackbars>', function () {
+  suite('base usage', function () {
+    test('normal toast is displayed', function (done) {
+      showSnackBar('test message', {duration: 1000});
+
+      assert.exists(document.querySelector('#paper-toast-container'), 'Toast container should exist');
+      assert.exists(document.querySelector('paper-toast'), 'Toast should exist');
+      assert.notExists(document.querySelector('paper-icon-button'), 'Toast close button should exist');
+
+      setTimeout(() => {
```

Comment:
Tests failed in travis in FF. I guess this is because you are using setTimeout to delay assertion. When you are doing that you should also instruct test runner how may last tests before timeout.
See here https://mochajs.org/#hook-level

Reply:
done